### ISSUE TITLE
Release EmojiTwo as an SVGinOT font

### ISSUE THREAD

**Issue by kwill** [EmojiTwo/emojitwo#162]
See https://github.com/eosrei/emojione-color-font for an example of this being done for Emoji One.

From https://github.com/eosrei/emojione-color-font/issues/71 :

> I'm willing to support EmojiTwo when there is a Unicode 10 release. The graphic design is a lot of work. Good luck! :D

**Comment by Crissov** [comment]
Yes, we still haven't found people who are willing and able to contribute the necessary artwork (and for free).

**Comment by survtur** [comment]
Why don't you make nightly (weekly or monthly) releases? You wrote that this process is fully automated.
